src/app/register-helper/register-helper.component.ts

```diff
@@ -63,8 +67,9 @@ export class RegisterHelperComponent implements OnInit {
     }
   }
   @ServerFunction({ allowed: true })
-  static async doHelperForm(args: any[], context?: Context) {
-    await executeOnServer(helperForm, args, context);
+  static async doHelperForm(args: any[],subject:string,message:string, context?: Context) {
+    await executeOnServer(helperForm, args, context);    
+    await EmailSvc.sendMail(subject, message.replace('!מתנדב!', args[0]), args[3]);
```

Comment:
this is the wrong place to call it - I would call it in the `doWork` method of the helper Form - it'll make more sense there and not require untyped calls such as args[0] etc...